net/unbound/Makefile

```diff
@@ -122,14 +122,14 @@ endef
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/unbound.h $(1)/usr/include/
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/include/unbound.h $(1)/usr/include/
 	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libunbound.{so*,a,la} $(1)/usr/lib/
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/lib/libunbound.{so*,a,la} $(1)/usr/lib/
```

Comment:
I thought having executable bit of shared library set is just a convention in UNIX systems, not required on Linux.  Anyway I am fine with either ways :)

By archive flag I mean `--archive` flag of `cp` command.

P.S. for archive purposes, the command for search for `.so` files with `x` bit set: `find /usr/lib -name '*.so*' -perm +0111 -type f`